content/how-to/cloud-servers/capture-packets-with-tcpdump/index.md

```diff
@@ -0,0 +1,76 @@
+---
+permalink: capture-packets-with-tcpdump/
+audit_date: '2020-10-12'
+title: Capture packets with tcpdump
+type: article
+created_date: '2013-04-25'
+created_by: Rose Contreras
+last_modified_date: '2020-10-12'
+last_modified_by: Rose Morales
+product: Cloud Servers
+product_url: cloud-servers
+---
+
+`tcpdump` is a powerful network debugging tool that you can use to intercept
+and display packets on a network interface. An important feature of `tcpdump`
+is the filter that enables you to display only the packets you want to see.
+
+### Install tcpdump
+
+This example uses Ubuntu&reg; 8.10, but the installation steps are
```

Comment:
Ubuntu 8.10 is insanely old. The SME didn't recommend updating the instructions to a newer version?